srv/_grains/platform.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from pathlib import Path
+
+FILENAME="/opt/seagate/lr-serial-number"
+
+
+def add_lr_serial_number():
+    """
```

Comment:
Need to fix this issue reported by codacy:
https://app.codacy.com/gh/Seagate/cortx-prvsnr/pullRequest?prid=7185140#newIssuesView